include/openssl/core_names.h

```diff
@@ -186,6 +186,7 @@ extern "C" {
 #define OSSL_KDF_PARAM_SECRET       "secret"    /* octet string */
 #define OSSL_KDF_PARAM_KEY          "key"       /* octet string */
 #define OSSL_KDF_PARAM_SALT         "salt"      /* octet string */
+#define OSSL_KDF_PARAM_CUSTOM       "custom"    /* utf8 string */
```

Comment:
Maybe use #define OSSL_ALG_PARAM_CUSTOM "custom"

#define OSSL_KDF_PARAM_CUSTOM OSSL_ALG_PARAM_CUSTOM

#define OSSL_MAC_PARAM_CUSTOM OSSL_ALG_PARAM_CUSTOM

Reply:
No need, this is gone now.